src/public/js/main.js

```diff
@@ -15,16 +17,33 @@
       }
 
       var msg = $('#post').val();
-      var encoded = RS.encode(msg);
+      var i = 1;
+      var last = 0;
+      var encoded = new Array(0);
+      for (; i <= msg.length; i++) {
+        if (i % k == 0) {
+          Array.prototype.push.apply(encoded, RS.encode(msg.slice(i - k, i)));
```

Comment:
Ok I remembered why I didn't get rid of it before! But, this would be a good application of the data structure slice array type thing I was talking about earlier in pseudo-talk!